experimental/PropertyDDS/packages/property-properties/api-extractor.json

```diff
@@ -1,4 +0,0 @@
-{
```

Comment:
Note: this package and the other propertyDDS packages in which the api-extractor configs were removed were not actually using api-extractor.